app/src/main/kotlin/fr/o80/mostwanted/data/data.kt

```diff
@@ -109,7 +109,7 @@ val exercises = listOf(
         id = 9,
         title = "Preview",
         explanation = { Explanation09() },
-        file = "E09Image.kt",
+        file = "S09Image.kt",
```

Comment:
C'est bien `E09Image.kt` je pense
```suggestion
        file = "E09Image.kt",
```